messages/admin.py

```diff
@@ -0,0 +1,4 @@
+# from django.contrib import admin
+# Remove comment when implementing admin
+
+# Register your models here.
```

Comment:
I think that you need to register your models

Reply:
fixed